ASIM/lib/functions/ASIM_LookupHTTPStatusCode.yaml

```diff
@@ -0,0 +1,128 @@
+Function:
+  Title: An ASIM function to returns the HTTP Status code name based on a numerical response code
+  Version: '0.1'
+  LastUpdated: Nov 18 2022
+References:
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+- Title: List of HTTP Status Codes
+  Link: https://en.wikipedia.org/wiki/List_of_HTTP_status_codes
+Description: |
+  This ASIM function returns the HTTP Status code name associated with the response code numerical value provided as a parameter. For example, for the Status code 200, the function returns OK.  
+FunctionName: ASIM_LookupHTTPStatusCode
+EquivalentBuiltInFunction: _ASIM_LookupHTTPStatusCode
+FunctionParams:
+  - Name: StatusCode
+    Type: int
+
+FunctionQuery: |
+let function = (StatusCode:int) {
```

Comment:
For YAML validation, the value of the "FunctionQuery" field from line 19 and on must be indented.